db/model/Variable.ts

```diff
@@ -37,15 +42,50 @@ export interface VariableRow {
     timespan?: string
     columnOrder?: number
     catalogPath?: string
-    dimensions?: string
+    dimensions?: Dimensions
 }
 
+interface EntityRow {
+    entityId: number
+    entityName: string
+    entityCode: string
+}
+
+interface Dimensions {
+    originalName: string
+    originalShortName: string
+    filters: {
+        name: string
+        value: string
+    }[]
+}
+
+type VariableQueryRow = Readonly<
+    UnparsedVariableRow & {
+        display: string
+        datasetName: string
+        nonRedistributable: number
+        sourceName: string
+        sourceDescription: string
+        dimensions: string
+    }
+>
+
 export type UnparsedVariableRow = VariableRow & { display: string }
 
 export type Field = keyof VariableRow
 
 export const variableTable = "variables"
 
+const initDuckDB = (): any => {
```

Comment:
do we not have types for `duckdb.Database` ?

Reply:
No 😭 There's only an **outdated** [node-duckdb ](https://github.com/deepcrawl/node-duckdb) library. I was thinking about adding my own, but given that we're still not sure whether to use duckdb, duckdb-wasm or something else, I didn't bother. (And that we call it only once)